pallets/issue/src/tests.rs

```diff
@@ -55,6 +57,23 @@ fn request_issue_ok_with_address(
 	Issue::_request_issue(origin, amount, vault).unwrap()
 }
 
+fn request_issue_ok_with_address_for_rate_limits(
+	origin: AccountId,
+	amount: Balance,
+	vault: DefaultVaultId<Test>,
+) -> Result<H256, DispatchError> {
+	ext::vault_registry::ensure_not_banned::<Test>.mock_safe(|_| MockResult::Return(Ok(())));
+
+	ext::security::get_secure_id::<Test>.mock_safe(|_| MockResult::Return(get_dummy_request_id()));
+
+	ext::vault_registry::try_increase_to_be_issued_tokens::<Test>
+		.mock_safe(|_, _| MockResult::Return(Ok(())));
+	ext::vault_registry::get_stellar_public_key::<Test>
+		.mock_safe(|_| MockResult::Return(Ok(DEFAULT_STELLAR_PUBLIC_KEY)));
+
+	Issue::_request_issue(origin, amount, vault)
+}
+
```

Comment:
Isn't this function just a duplicate of `request_issue_ok_with_address()`? If so, I'd remove it.

Reply:
the same but without `unwrap`
`Issue::_request_issue(origin, amount, vault)`